error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJRules.java

```diff
@@ -162,35 +159,6 @@ OptionalLongAssert after(OptionalLong optional, long expected) {
     }
   }
 
-  //
```

Comment:
Don't we want to keep this comment?

Reply:
Ah, I thought it was the only rule in this section; reinstated it. :+1: